web/src/locales/en.json

```diff
@@ -35,11 +35,11 @@
     "title": "Title",
     "filter": "Filter",
     "tags": "Tags",
-    "yourself": "Yourself",
+    "yourself": "You",
```

Comment:
We should use the same key and value in locale. So let's add a new field `"you": "You"` instead of changing `"yourself"` in a separate PR. 

So it's better to fix the existed typo in this PR. e.g. `passworld`